data/language/english_uk.txt

```diff
@@ -885,7 +885,7 @@ STR_0879    :Can't lower land here...
 STR_0880    :Can't raise land here...
 STR_0881    :Object in the way
 STR_0882    :Load Game
-STR_0883    :Save Game
+STR_0883    :Save Game as
```

Comment:
I'll change it. I had to choose between changing it in the code or changing it in the language files...